Checkstyle failed:

```
22:31:45 [ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk11-scala2.13/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java:194:40: ')' is preceded with whitespace. [ParenPad]
```

--- a/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java
+++ b/connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java
@@ -191,7 +191,7 @@ public class MirrorConnectorsIntegrationTest {
         for (String connector : connNames) {
             TestUtils.waitForCondition(() -> areConnectorAndTasksRunning(connectCluster,
                 connector), "Timed out trying to verify connector " +
-                connector + " was up!" );
+                connector + " was up!");
         }
     }
 

